lib/galaxy/util/watcher.py

```diff
@@ -71,11 +71,22 @@ def __init__(self, observer_class, even_handler_class, **kwargs):
         self.observer = None
         self.observer_class = observer_class
         self.event_handler = even_handler_class(self)
+        self.monitored_dirs = {}
 
     def start(self):
         if self.observer is None:
             self.observer = self.observer_class()
             self.observer.start()
+            self.resume_watching()
+
+    def monitor(self, dir, recursive=False):
```

Comment:
`dir` is a standard library built-in function.

```suggestion
    def monitor(self, dir_path, recursive=False):
```